bundles/org.openhab.binding.flume/README.md

```diff
@@ -0,0 +1,120 @@
+# Flume Binding
+
+This binding will interface with the cloud API to retrieve water usage from your [Flume](https://flumewater.com/) water monitor.
+
+## Introduction
+
+The Cloud Connector is required as a "bridge" to interface to the cloud service from Flume.
+While the Flume API supports a rich querying of historical usage data, this binding only retrieves the cumulative water used and instantaneous water used, thus relying on Openhab's rich persistence services for exploring historical values.
+
+## Supported Things
+
+This binding supports the following things:
+
+| thing                     | type          | description                  |
+|----------                 |--------       |------------------------------|
+| Flume Cloud Connector     | Bridge        | This represents the cloud account to interface with the Flume API.  |
+| Flume Meter Device        | Device        | This interfaces to a specific Flume water monitor associated with the account. |
```

Comment:
Could make perfectly sense, but just to make sure this is wellthought chosen id:
Flume might extend their product range, or add new ranges all supported by this API. Woudl it still make senso to have the thing called `device`. If a new range supports data that the current devices don't, how would this be modeled.?

Reply:
good suggestion.